### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by christophvogel**
- Fixed issues from my last PR
- Updated globals.css
- Deployment: https://refcast-pipe-refactor-9kty3c8v.herokuapp.com/

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
Good Job! 🚀

**Review by cboenke — APPROVED** [review]
LGTM 🚀

File: pages/index.tsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ export default function Home() {
       </Head>
 
       <main className={styles.main}>
-          <Logo size="big"></Logo>
-          <LoginButton></LoginButton>
-          <SkipLoginButton></SkipLoginButton>
+        <Logo size="big"></Logo>
+        <LoginButton></LoginButton>
+        <SkipLoginButton></SkipLoginButton>
```
Comment (fre-ben) [inline pages/index.tsx:18]: You could write it shorter like this, no need for closing the Buttons like you do, if they don't have child elements:
`<Logo />
<LoginButton />
<SkipLoginButton />`

**Review by fre-ben — CHANGES_REQUESTED** [review]
LGTM, just a minor change 🧙

**Pull request merged**